scripts/new_project.kts

```diff
@@ -1,25 +1,68 @@
+#!/usr/bin/env kscript
+
+@file:MavenRepository("central", "https://repo.maven.apache.org/maven2/")
+@file:DependsOn("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.1")
+
+import kotlinx.coroutines.*
+import java.io.File
+import kotlin.coroutines.coroutineContext
```

Comment:
Could we sort the imports alphabetically? It will look a bit more organized that way 😁

Reply:
@kaungkhantsoe Oops! I believe it should be like this if we want to sort it alphabetically, right?

```
import java.io.File
import kotlin.coroutines.coroutineContext
import kotlinx.coroutines.*
``